web/packages/teleterm/src/services/pty/ptyHost/buildPtyOptions.ts

```diff
@@ -129,14 +137,23 @@ export function getPtyProcessOptions(
         ? `${cmd.login}@${cmd.serverId}`
         : cmd.serverId;
 
+      const argsResume = [
+        `--proxy=${cmd.rootClusterId}`,
+        'ssh',
+        '--forward-agent',
+        loginHost,
+      ];
+      const argsNoResume = [
+        `--proxy=${cmd.rootClusterId}`,
+        'ssh',
+        '--no-resume',
+        '--forward-agent',
+        loginHost,
+      ];
```

Comment:
Nit: I think we could use a single arguments array to avoid duplication.
```suggestion
      const args = [
        `--proxy=${cmd.rootClusterId}`,
        'ssh',
        sshOptions.noResume && '--no-resume',
        '--forward-agent',
        loginHost,
      ].filter(Boolean);
```

Reply:
That will also remove an empty `loginHost` tho, right? If somehow `loginHost` is empty, we'll show the full usage for `tsh ssh` (probably bad) but end with `ERROR: required argument '[user@]host' not provided` instead of erroring out with `ERROR: failed connecting to host :0: failed to receive cluster details response` (good).

Reply:
Yeah, if for some reason `loginHost` was an empty string, it would be filtered out. To fix this, we can do the following:
```ts
      const args = [
        `--proxy=${cmd.rootClusterId}`,
        'ssh',
        sshOptions.noResume ? '--no-resume' : null,
        '--forward-agent',
        loginHost,
      ].filter(i => i !== null);
```

Reply:
The spread operator would work here as well and let us avoid a special `filter`. https://2ality.com/2017/04/conditional-literal-entries.html

Reply:
I went with that in 345bcb7, I like it.